### DISCUSSION TITLE
rdpSettings::MouseUseRelativeMove

### DISCUSSION THREAD

**Discussion by jfrgagnon** [FreeRDP/FreeRDP#8322] [Q&A]
I'd like to know how to activate the mouse relative function instead of absolute?

Here's my command : 
xfreerdp /u:XXXXXX /p:XXXXX /sound /microphone /rfx /rfx-mode:video /gfx /nsc /dynamic-resolution /workarea /v:1**.1**.1**.***:3*** -compression +gestures +clipboard

But still the only missing part is the mouse relative input... :\ Thanks for your help!

**Comment by akallabeth** [comment]
1. You are using a development version of `FreeRDP` which has the `ainput` channel build in
1. you have added `+mouse-relative` to your command line.
1. Will only work with servers supporting the channel though (currently only gnome-remote-desktop is implementing that as far as I know)
